tests/test_ops.py

```diff
@@ -20,7 +20,7 @@
 from devito.ops.utils import namespace # noqa
 from devito.symbolics import Byref, Literal, indexify # noqa
 from devito.tools import dtype_to_cstr # noqa
-from devito.types import Constant, Symbol # noqa
+from devito.types import Constant, Symbol, Buffer # noqa
```

Comment:
Just for coherence Devito usually sort the import alphabetically, so here `Buffer` should be ther first one. 
But as there is thi s` # noqa`. I don't know if its necessary.

PS: `indexify` should also change its position but I know it was probably me that did this.

Reply:
thank you, fixed